doc/man/8/ceph-volume.rst

```diff
@@ -34,6 +36,25 @@ them.
 Commands
 ========
 
+inventory
+---------
+
+This subcommand provides information about a hosts physical disc inventory and
+reports metadata about these discs. Among this metadata one can find disc
+specific data items (like model, size, rotational or solid state) as well as
+data items specific to ceph using a device (available for use with ceph, present
+logical volumes a.o.).
+
+Examples::
+    ceph-volume inventory
+    ceph-volume inventory /dev/sda
+    ceph-volume inventory --format json-pretty
+
+Optional arguments:
```

Comment:
The docs build is complaining about line 56 being unexpectedly indented:

> /home/jenkins-build/build/workspace/ceph-pr-docs/doc/man/8/ceph-volume.rst:56:Unexpected indentation.

The error is a bit misleading, I think the indentation is correct, just that there isn't a new line after `Optional arguments:` which causes the `*` not to be rendered as a bullet list.

```suggestion
Optional arguments:

```